src/binder/clrprivbindercoreclr.cpp

```diff
@@ -159,8 +159,8 @@ HRESULT CLRPrivBinderCoreCLR::BindUsingPEImage( /* in */ PEImage *pPEImage,
                 {
                     if (pCoreCLRFoundAssembly->GetIsInGAC())
```

Comment:
Do we need this GetIsInGAC check any more @jkotas? I think we should just remove it.